libs/coin-modules/coin-mina/.unimportedrc.json

```diff
@@ -0,0 +1,64 @@
+{
+  "entry": [
+    "src/account.ts",
+    "src/bridge/js.ts",
+    "src/cli-transaction.ts",
+    "src/deviceTransactionConfig.ts",
+    "src/errors.ts",
+    "src/hw-getAddress.ts",
+    "src/serialization.ts",
+    "src/specs.ts",
+    "src/transaction.ts"
+  ],
+  "ignoreUnimported": [
+    "src/api/index.ts",
+    "src/api/rosetta/index.ts",
+    "src/api/rosetta/types.ts",
+    "src/api/rosetta/utils.ts",
+    "src/bridge/broadcast.ts",
+    "src/bridge/buildTransaction.ts",
+    "src/bridge/cli-transaction.ts",
+    "src/bridge/createTransaction.ts",
+    "src/bridge/deviceTransactionConfig.ts",
+    "src/bridge/errors.ts",
+    "src/bridge/estimateMaxSpendable.ts",
+    "src/bridge/getFeesForTransaction.ts",
+    "src/bridge/getTransactionStatus.ts",
+    "src/bridge/index.ts",
+    "src/bridge/prepareTransaction.ts",
+    "src/bridge/signOperation.ts",
+    "src/bridge/synchronisation.ts",
+    "src/bridge/transaction.ts",
+    "src/common-logic/index.ts",
+    "src/config.ts",
+    "src/consts.ts",
+    "src/signer/getAddress.ts",
+    "src/signer/index.ts",
+    "src/test/bot-specs.ts",
+    "src/test/bridgeDatasetTest.ts",
+    "src/test/index.ts",
+    "src/test/speculos-deviceActions.ts",
+    "src/test/testUtils.ts",
+    "src/types/common.ts",
+    "src/types/index.ts",
+    "src/types/signer.ts"
+  ],
+  "ignoreUnresolved": [],
+  "ignoreUnused": [
+    "@ledgerhq/coin-framework",
```

Comment:
If it's unused it needs to be removed from package.json

you can't just add them all here

Reply:
I see, it should be better now. Please have another look 🙏🏻